events/2016/1-nodeschool.md

```diff
@@ -1,7 +1,8 @@
 ---
 name: Nodeschool
+slug: nodeschool
```

Comment:
Good shout - this did feel a bit ugly. I'll generate a `_path` key on the event or something.